Dev/src/study_tools/build_index.py

```
@@ -0,0 +1,67 @@
+        text = "\n\n".join(doc[pg].get_text() for pg in range(i, end))
+        meta = {
+            "file_path": str(pdf_path),
+            "file_name": pdf_path.name,
+            "page_start": i + 1,
+            "page_end": end,
+        }
+        yield Document(text=text, metadata=meta)
+
+
+def main():
+    from llama_index.core import VectorStoreIndex, StorageContext, Document
```

Flake8 (7.2.0)

[error] 'llama_index.core.Document' imported but unused

(F401)